project-rome-docs/objectivec-api/remotesystems.commanding/MCDAppServiceRequestReceivedEventArgs.md

```diff
@@ -0,0 +1,19 @@
+---
+title: MCDAppServiceRequestReceivedEventArgs
+description: Contains data associated with a "request received" event.
+keywords: microsoft, windows, iOS, iPhone, objectiveC, connected devices, Project Rome
+---
+
+# class `MCDAppServiceRequestReceivedEventArgs` 
+
+```
+@interface MCDAppServiceRequestReceivedEventArgs : NSObject
+```  
+Contains data associated with a "request received" event.
+
+## Properties
+
+### request
+`@property(nonatomic, readonly, nonnull) MCDAppServiceRequest* request;`
+
+The connection request associated with the target device.
```

Comment:
The request sent by the remote device.

It's not a connection request -- the connection has already been established at this point.